redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseView.kt

```diff
@@ -22,22 +22,19 @@ import kotlinx.coroutines.flow.StateFlow
 import kotlinx.serialization.json.Json
 
 public interface TreehouseView<A : AppService> {
```

Comment:
I think we can eliminate this `A`, or are at least really close to it. It's still used a bunch but I think that's only a stand-in for it being present on other types or simply a wildcard.

Reply:
Yeah, that’s my next PR.